pylab/website/views.py

```diff
@@ -103,22 +105,25 @@ def create_weekly_event(request, year, month, day, slug):
 def voting_page(request, voting_poll_slug):
     voting_poll = get_object_or_404(VotingPoll, slug=voting_poll_slug)
     total_points = 15
-    VotePointsFormSet = modelformset_factory(
-        Vote,
-        form=website_forms.VotePointsForm,
+    ProjectPointsFormSet = modelformset_factory(
+        Project,
+        form=website_forms.ProjectPointsForm,
         formset=website_forms.BaseTotalPointsFormset,
         extra=0,
     )
-    vote_qs = Vote.objects.filter(voter=request.user, voting_poll__slug=voting_poll_slug)
+    ProjectPointsFormSet.form = staticmethod(curry(
+        website_forms.ProjectPointsForm,
+        user=request.user,
+        voting_poll=voting_poll))
```

Comment:
That looks like a dark magic, but this is more related with FormSet limitation, probably, I will look into it later, when I have more time. I'm sure, that there can be better solution.